test/main.js

```diff
@@ -210,6 +210,7 @@ describe('node-lambda', function () {
       fs.mkdirsSync(path.join('__unittest', 'hoge'));
       fs.mkdirsSync(path.join('__unittest', 'fuga'));
       fs.writeFileSync(path.join('__unittest', 'hoge', 'piyo'));
+      fs.writeFileSync(path.join('__unittest', 'hoge', 'package.json'));
```

Comment:
The setting to exclude `__unittest/hoge/*` is https://github.com/motdotla/node-lambda/pull/274/files#diff-e5b8495db5b34b144973178386ba5ceeR247.